certified-connectors/Peltarion/apiProperties.json

```diff
@@ -13,10 +36,36 @@
           "x-ms-apimTemplateParameter.name": "content-type",
           "x-ms-apimTemplateParameter.value": "application/json",
           "x-ms-apimTemplateParameter.existsAction": "override",
-          "x-ms-apimTemplate-policySection": "Request",
-          "x-ms-apimTemplate-operationName": [
-            "callapi"
-          ]
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      },
+      {
+        "templateId": "setqueryparameter",
+        "title": "Query token",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "code",
+          "x-ms-apimTemplateParameter.value": "FHWJvOJnrrBz8v7DOWoABLS1K2e9Im6q7YalAthQMGkqTa57jDYm/w==",
```

Comment:
@andvra It seems to be some shared key to access your service. 
1) Please note that this key is publicly visible to anyone.
2) You may place into connection parameters (as url and token) and share the key with your customers explicitly

Reply:
Thanks for noting. The code isn't needed, so removed it from the code now.